hermes-management/src/test/groovy/pl/allegro/tech/hermes/management/infrastructure/metrics/HybridSubscriptionMetricsRepositoryTest.groovy

```diff
@@ -42,5 +46,26 @@ class HybridSubscriptionMetricsRepositoryTest extends Specification {
         metrics.delivered == 100
         metrics.discarded == 1
         metrics.inflight == 5
+        metrics.timeoutsM1 == "100"
+        metrics.otherErrorsM1 == "1000"
+    }
+
+    def "should read subscription metrics for all http status codes"() {
+        def codeClasses = ['sumSeries(stats.consumer.*.status.group.topic.subscription.1xx.m1_rate)', 'sumSeries(stats.consumer.*.status.group.topic.subscription.2xx.m1_rate)',
+                           'sumSeries(stats.consumer.*.status.group.topic.subscription.3xx.m1_rate)', 'sumSeries(stats.consumer.*.status.group.topic.subscription.4xx.m1_rate)',
+                           'sumSeries(stats.consumer.*.status.group.topic.subscription.5xx.m1_rate)']
+
+        given:
+        client.readMetrics(codeClasses[0]) >> new GraphiteMetrics().addMetricValue(codeClasses[0], '0')
```

Comment:
maybe there could be some private method like 'graphiteTargetForStatusCodeFamily("2xx")` returning that string instead of an indexed array